quad_logger/scripts/align_Ylabels.m

```diff
@@ -0,0 +1,151 @@
+function align_Ylabels(figureHandle)
+%ALIGN_YLABELS - Align the y-labels in the each column of a subplots-based figure such as they all
+%have the same x-coordinate value. The function works both with the default matlab subplot command
+%and with the subplot1 (http://www.mathworks.com/matlabcentral/fileexchange/9694) function.
+%Rescaling the figure after running the command messes up with the ylabels. Maybe I'll take a look
+%to that in the future (but not necessarily...). It works also when multiline labels are used.
+%
+%At row 93, I use the tilde for an unused argument. I'm pretty sure that kind of syntax is not

```

Comment:
Are these comments meant to be merged?

Reply:
Not my function, came from matlab file exchange. Do we want to edit it at all or just leave as is?